### ISSUE TITLE
Converter tests failing on TF 1.14.0

### ISSUE THREAD

**Issue by mortendahl** [tf-encrypted/tf-encrypted#634]
Bug can be reproduced by simply running the tests (`python -m pytest -k convert_test`) with TF 1.14.0 installed.

**Comment by yanndupis** [comment]
I have started to investigate these bugs. 

**None deterministic test**
The converter tests fails for keras layers because even if we have set the [tf seed], for some reasons the tf layer doesn't returns the same output after the layer get restored from the exported graph. Setting the seed in the kernel initializer solve the problem. For example:
```
 d = Dense(2,
       kernel_initializer=tf.keras.initializers.RandomNormal(seed=1),
       input_shape=shape[1:])
```

**Can't export the Keras batchnorm layer to a pb file**
In tf 1.14, they have introduced this[ new check](https://github.com/tensorflow/tensorflow/blob/456fbc0e498e3d10604973de9f46ca48d62267cc/tensorflow/python/framework/graph_util_impl.py#L246) in `graph_util.convert_variables_to_constants`. It raises an error when we export the graph to a pb file. 

Not sure yet how to fix this.

**Comment by justin1121** [comment]
> Can't export the Keras batchnorm layer to a pb file

Interestingly this is different than whats on master: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/framework/graph_util_impl.py

**Comment by yanndupis** [comment]
ah nice find @justin1121! Maybe we could have a copy of the graph utils file (Master or tf 1.13 version) and call `convert_variables_to_constants` from it. 

What do you guys think?

**Comment by justin1121** [comment]
This is done.
